### PR TITLE
[tensorflow][test][ec2]Adding Local mode for running ec2 test

### PR DESCRIPTION
**Pull request by Elizaaaaa**
*Description:*
When running DLC tests in TF release pipeline, we met issues of creating relevant resources under the aws tf account. In order to run ec2 tests directly in containers on tf pipeline, I'm adding this --local-mode flag to skip remotely running tests on ec2 instances .

*Tests run:*
Tests passed on tf pipeline.

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by YYStreet — DISMISSED** [review]

**Comment by tejaschumbalkar** [comment]
@Elizaaaaa can you resolve the file conflicts?

File: test/test_utils/ec2.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from botocore.config import Config
 from botocore.exceptions import ClientError
 
-from . import DEFAULT_REGION, UL_AMI_LIST, LOGGER, BENCHMARK_RESULTS_S3_BUCKET
+from . import DEFAULT_REGION, UL_AMI_LIST, LOGGER, CONTAINER_TESTS_PREFIX, BENCHMARK_RESULTS_S3_BUCKET
```
Comment (arjkesh) [inline test/test_utils/ec2.py:12]: any reason for this addition? looks like it is causing a merge conflict

**Comment by arjkesh** [comment]
@Elizaaaaa moving this to draft for now until you have a chance to make updates - the merge conflict can likely be resolved by moving the CONTAINER_TESTS_PREFIX import, which appears unused.

**Comment by saimidu** [comment]
@Elizaaaaa What needs to be done to get this PR to a Review-Ready state?

**Review by SergTogul — APPROVED** [review]

**Comment by arjkesh** [comment]
Hello - this PR has not been accessed for awhile. We are closing it for housekeeping purposes - please reopen if needed.

**Pull request closed without merging**